Code/Framework/AzCore/AzCore/Debug/LocalFileEventLogger.cpp

```diff
@@ -246,7 +246,7 @@ namespace AZ::Debug
         {
         }
 
-        uint32_t writeSize = AZ_SIZE_ALIGN_UP(sizeof(EventHeader) + size, EventBoundary);
+        uint32_t writeSize = AZ_SIZE_ALIGN_UP(sizeof(EventHeader) + size, Metrics::EventBoundary);
```

Comment:
nit: Earlier in the file you used `Metrics::IEventLogger::EventHeader`, should keep it consistent since these changes center around namespace qualifications.

Reply:
Actually it is inconsistent for a reason.
`EventHeader` is allowed to be used functions that are within the `LocalFileEventLogger` scope since it inherits from `IEventLogger`.

But if you notice here `Metrics::IEventLogger::EventHeader` has to be used here because the scope is within the `EventLogReader` class which is unrelated.